README.md

```
@@ -1,41 +1,339 @@
+   Docker will pull the pre-built `ubuntu-24.04-userver` image (~6 GB, one-time download).
+
+2. Configure and build:
+
+```bash
+make build-debug     # Debug build with AddressSanitizer + UBSan
+make build-release   # Optimized release build
+```
+
+> `secdist.json` is created automatically from `secdist.example.json` on first run.
+
+3. Run tests:
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)